src/MICore/Debugger.cs

```diff
@@ -55,6 +55,24 @@ public bool IsClosed
             }
         }
 
+        public bool IsDebuggerRunning
```

Comment:
This method is a bit weird in that it doesn't do what it says it does on Windows. That is fine for how it is being used now, but its not very obvious for the future. I would suggest -
* private instead of public
* A method instead of a property
* Something that indicates that it doesn't doing anything on Windows: IsUnixDebuggerRunning?